codes/csharp/chapter_dynamic_programming/min_path_sum.cs

```diff
@@ -0,0 +1,126 @@
+﻿/**
+* File: min_path_sum.cs
+* Created Time: 2023-07-10
+* Author: hpstory (hpstory1024@163.com)
+*/
+
+namespace hello_algo.chapter_dynamic_programming; 
+
+public class min_path_sum {
+    /* 最小路径和：暴力搜索 */
+    public int minPathSumDFS(int[][] grid, int i, int j) {
+        int INF = 0x3f3f3f3f;
+        // 若为左上角单元格，则终止搜索
+        if (i == 0 && j == 0){
+            return grid[0][0];
+        }
+        // 若行列索引越界，则返回 +∞ 代价
+        if (i < 0 || j < 0) {
+            return INF;
+        }
+        // 计算从左上角到 (i-1, j) 和 (i, j-1) 的最小路径代价
+        int left = minPathSumDFS(grid, i - 1, j);
+        int up = minPathSumDFS(grid, i, j - 1);
+        // 返回从左上角到 (i, j) 的最小路径代价
+        return Math.Min(left, up) + grid[i][j];
+    }
+
+    /* 最小路径和：记忆化搜索 */
+    public int minPathSumDFSMem(int[][] grid, int[][] mem, int i, int j) {
+        int INF = 0x3f3f3f3f;
```

Comment:
I don’t know how to modify this. In C#, int.MaxValue should not be used directly, as there may be an overflow in the following addition. Usually, 0x3f is used instead. Should I use int.MaxValue directly or explain why 0x3f is used?